### ISSUE TITLE
Is not bringing data requested

### ISSUE THREAD

**Issue by chachoe** [restfb/restfb#864]
Hi! I´m using restfb since a year ago. Just now my app is breaking because when i´m fetching reactions and comments from a post, the restfb is not bringing the data i requested.
For example, for a post i´m doing the following:
```
// Obtein the reactions from thepost
Connection<ReactionItem> reacciones = fbClient.get().fetchConnection(idPost + "/reactions",
		ReactionItem.class, Parameter.with("fields", "id,name,type"));

// for each one
for (List<ReactionItem> reaccionesConnectionPage : reacciones) {
	for (ReactionItem reaccion : reaccionesConnectionPage) {

		// here i do the logic. but now never im getting here
	}
```
The list of the reactions is always blanck, with no data. 

The same with the comments:
```
Connection<Comment> commentConnection = fbClient.get().fetchConnection(idPost + "/comments", Comment.class,
					Parameter.with("metadata", 1));

// Para cada comentario
for (List<Comment> commentPage : commentConnection) {
	for (Comment comment : commentPage) {
		//Here the logic
	}
}
```

Also i cant reach the information im requesting.. Doy you know why? 
This is happening since februrary.

**Comment by nbartels** [comment]
Great, that your app is based on RestFB. We really like this kind of feedback.

Too bad, the problem you describe is Facebook based. In February 2018 they changed the behaviour of the reactions. You can only fetch them with a page access token. Check the reference here: https://developers.facebook.com/docs/graph-api/reference/v2.12/object/reactions

In the permissions section, you'll find the hint. 

Facebook says: 

> On February 5th, 2018, User information will not be included in responses unless you make the request with a Page access token. This only applies to Comments on Pages and Posts on Pages.

**Comment by chachoe** [comment]
Thanks for the answer! So, now I need the page acces token to fetch the information im asking.. Can you code me how to do that?
Is in this way?

```
// Obtein the reactions from thepost
Connection<ReactionItem> reacciones = fbClient.get().fetchConnection(idPost + "/reactions",
		ReactionItem.class, Parameter.with("acces-token", MY_PAGE_TOKEN
                                                                 .with("fields", "id,name,type"));

// for each one
for (List<ReactionItem> reaccionesConnectionPage : reacciones) {
	for (ReactionItem reaccion : reaccionesConnectionPage) {

		// now i will reach this information?
	}
```

**Comment by nbartels** [comment]
no. Create a new DefaultFacebookClient with your MY_PAGE_TOKEN. And use that client to do the request. So I think you only need to replace the user access facebook client with the page access facebook client. The rest of your implementation should stay the same.
